# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../flightgear/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur FlightGear-0.9.9-orig/src/Environment/environment_ctrl.hxx FlightGear-0.9.9/src/Environment/environment_ctrl.hxx
--- FlightGear-0.9.9-orig/src/Environment/environment_ctrl.hxx	2005-11-17 22:02:00.000000000 +0000
+++ FlightGear-0.9.9/src/Environment/environment_ctrl.hxx	2006-04-04 17:49:50.537343000 +0000
@@ -259,4 +259,6 @@
     double _error_dt;
 };
 
+void metar_cleanup_handler( void* );
+
 #endif // _ENVIRONMENT_CTRL_HXX
diff -Nur FlightGear-0.9.9-orig/src/Environment/fgclouds.hxx FlightGear-0.9.9/src/Environment/fgclouds.hxx
--- FlightGear-0.9.9-orig/src/Environment/fgclouds.hxx	2005-10-26 14:15:42.000000000 +0000
+++ FlightGear-0.9.9/src/Environment/fgclouds.hxx	2006-04-04 17:32:43.145135000 +0000
@@ -53,7 +53,7 @@
 
 	void update_metar_properties( FGMetar *m );
 
-	void FGClouds::update_env_config ();
+	void update_env_config ();
 
 	int update_event;
 	SGSoundSample *snd_lightning;
diff -Nur FlightGear-0.9.9-orig/src/Scenery/FGTileLoader.hxx FlightGear-0.9.9/src/Scenery/FGTileLoader.hxx
--- FlightGear-0.9.9-orig/src/Scenery/FGTileLoader.hxx	2004-06-15 03:26:03.000000000 +0000
+++ FlightGear-0.9.9/src/Scenery/FGTileLoader.hxx	2006-04-04 18:05:02.470335250 +0000
@@ -164,4 +164,6 @@
 #endif // ENABLE_THREADS
 };
 
+void cleanup_handler( void* );
+
 #endif // FG_TILE_LOADER_HXX
diff -Nur FlightGear-0.9.9-orig/src/Traffic/SchedFlight.hxx FlightGear-0.9.9/src/Traffic/SchedFlight.hxx
--- FlightGear-0.9.9-orig/src/Traffic/SchedFlight.hxx	2005-10-26 14:15:43.000000000 +0000
+++ FlightGear-0.9.9/src/Traffic/SchedFlight.hxx	2006-04-04 17:47:53.250013000 +0000
@@ -67,7 +67,7 @@
   FGScheduledFlight();
   FGScheduledFlight(const FGScheduledFlight &other);
   //  FGScheduledFlight(const string);
-  FGScheduledFlight::FGScheduledFlight(const string& cs,
+  FGScheduledFlight(const string& cs,
 		     const string& fr,
 		     const string& depPrt,
 		     const string& arrPrt,
